### PR TITLE
Add GluonFX plugin to Maps sample

### PR DESCRIPTION
**Pull request by abhinayagarwal**
This PR enables building and running native image for Maps sample

### CODE REVIEW

File: samples/mobile/README.md [hunk 1]
```diff
@@ -2,12 +2,55 @@
 
 This sample demonstrates how to use Gluon Maps inside a JavaFX application.
 
-
 ## Instructions
 
-To execute this sample, using Java 11, do as follows:
+We use [GluonFX plugin](https://docs.gluonhq.com/) to build a native image for platforms including desktop, android and iOS.
```
Comment (jperedadnr) [inline samples/mobile/README.md:7]: Also embedded (not that you need to add instructions)

File: samples/mobile/build.gradle [hunk 2]
```diff
@@ -39,15 +40,24 @@ repositories {
...
 }
 
+gluonfx {
+    target = 'host'
+    if (project.hasProperty('target')) {
+        target = project.getProperty('target')
+    }
+    attachConfig {
+        version = "$attachVersion"
+        configuration = "implementation";
```
Comment (jperedadnr) [inline samples/mobile/build.gradle:57]: isn't it the default configuration already?

File: samples/mobile/src/main/java/com/gluonhq/maps/samples/mobile/MobileSample.java [hunk 3]
```diff
@@ -39,6 +39,10 @@
 import javafx.geometry.Pos;
 import javafx.geometry.Rectangle2D;
 import javafx.scene.Scene;
+import javafx.scene.control.Label;
```
Comment (jperedadnr) [inline samples/mobile/src/main/java/com/gluonhq/maps/samples/mobile/MobileSample.java:42]: update license header

File: build.gradle [hunk 4]
```diff
@@ -33,7 +33,7 @@ plugins {
 
 apply plugin: 'base'
 
-ext.attachVersion = "4.0.5"
+ext.attachVersion = "4.0.11"
```
Comment (jperedadnr) [inline build.gradle:36]: This is part of #56 as well, that should get integrated before this one.

Comment (abhinayagarwal) [reply build.gradle:36]: Sure, go ahead?

Comment (jperedadnr) [reply build.gradle:36]: maybe integrate #58 first?

Comment (abhinayagarwal) [reply build.gradle:36]: done

**Review by jperedadnr — APPROVED** [review]

**Pull request merged**